### DISCUSSION TITLE
Is ESP32_WROOM_32 nanoframework out of date ?

### DISCUSSION THREAD

**Discussion by MalcolmGillott** [nanoframework/Home#1015] [Q&A] [answered]
I am unable to find a combination of VS extensions and firmware that work for an ESP32_WROOM_32  module

I have Visual Studio Community Edition 2022 fully updated
Nanoframework extension 2022.2.0.8
nanoff installed firmware version 1.7.0.618 for ESP32_WROOM_32 on the module- this version built August 2021
When trying to deploy / debug "Hello World", I get a deployment error due to a mscorlib mismatch : -

1>The connected target has the wrong version for the following assembly(ies):
1>
1>    'mscorlib' requires native v100.5.0.17, checksum 0x004CF1CE.
1>    Connected target has v100.5.0.12, checksum 0x132BDB6F.

Is this easily fixable ?
Thanks
Malc

**Comment by josesimoes** [comment, marked as answer]
That target has been discontinued a long time ago. If you're uisng a recent version of nanoff you should have been presented a warning about this.

Please check the existing target names in the repo readme or use `--platform esp32` option instead of specifying the target name.

**Reply by Maumtrasna** [reply]
Well, I scratched my head reading the nanoframework targets page that says:

ESP32_REV0 that fits **all ESP32 boards carrying an ESP32 chip without PSRAM support**. These can be ESP32-DevKitC, **ESP32-WROOM** and ESP32-WROVER variants without PSRAM.

Am I missing something here? Is ESP-WROOM-32 (marked exactly that way) supported?

Markku

**Reply by josesimoes** [reply]
I say: go for it! 😁
Nevertheless, as mentioned a bit above if you use `-platform esp32` nanoff will choose the appropriate fw to flash the module with.
